packages/cli/src/commands/test.ts

```diff
@@ -50,7 +51,7 @@ export default class Test extends AuthCommand {
       default: [],
     }),
     'env-file': Flags.string({
-      description: 'dotenv file path to be passed.',
+      description: 'dotenv file path to be passed. For example --env-file="/.env"',
```

Comment:
@tnolet , is it OK to show an example with an absolute path `/.env`? Shoudn't it be `./.env`?